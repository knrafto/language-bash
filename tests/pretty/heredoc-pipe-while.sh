cat <<EOF1 | while true; do true; done
Here doc 1
EOF1
