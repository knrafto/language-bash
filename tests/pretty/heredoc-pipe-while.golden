cat <<EOF1 |
Here doc 1
EOF1
while true; do
    true;
done;
